.github/workflows/macOS-amd64.yml

```diff
@@ -0,0 +1,245 @@
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@v2
+        with:
+          name: ${{ needs.tensorflow_opt-macOS.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+        if: needs.tensorflow_opt-macOS.outputs.status == 'missing'
+      - uses: ./.github/actions/check_artifact_exists
+        with:
+          name: ${{ needs.tensorflow_opt-macOS.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+          download: true
+        if: needs.tensorflow_opt-macOS.outputs.status == 'found'
+      - run: |
+          tar xkf ${{ github.workspace }}/home.tar.xz
+          rm ${{ github.workspace }}/home.tar.xz
+      - run: |
+          git status
+      - uses: ./.github/actions/select-xcode
+        with:
+          version: "12.1.1"
+      - uses: ./.github/actions/host-build
+        with:
+          flavor: ${{ matrix.build-flavor }}
+      - uses: ./.github/actions/package
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.tar.xz"
+          path: ${{ github.workspace }}/artifacts/native_client.tar.xz
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "libdeepspeech.${{ matrix.build-flavor }}.zip"
+          path: ${{ github.workspace }}/artifacts/libdeepspeech.zip
+  build-python-macOS:
+    name: "Build python bindings for macOS"
+    runs-on: macos-10.15
+    needs: [ build-lib_macOS, swig_macOS ]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+        python-version: [3.6.8, 3.7.9, 3.8.8, 3.9.2]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/download-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.tar.xz"
+          path: ${{ github.workspace }}/tensorflow/bazel-bin/native_client/
+      - run: |
+          cd ${{ github.workspace }}/tensorflow/bazel-bin/native_client/
+          tar xf native_client.tar.xz
+          ls -hal
+          cd ${{ github.workspace }}/
+      - uses: actions/download-artifact@v2
+        with:
+          name: "swig_macOS"
+          path: ${{ github.workspace }}/native_client/ds-swig/
+      - run: |
+          ls -hal ${{ github.workspace }}/native_client/ds-swig/bin
+          ln -s ds-swig ${{ github.workspace }}/native_client/ds-swig/bin/swig
+          chmod +x ${{ github.workspace }}/native_client/ds-swig/bin/ds-swig ${{ github.workspace }}/native_client/ds-swig/bin/swig
+      - uses: ./.github/actions/install-python-upstream
+        with:
+          version: ${{ matrix.python-version }}
+      # GitHub packaged version are limited to macOS deployment target 10.14
+      #- uses: actions/setup-python@v2
+      #  with:
+      #    python-version: ${{ matrix.python-version }}
+      - uses: ./.github/actions/python-build
+        with:
+          project_name: deepspeech-${{ matrix.build-flavor }}
```

Comment:
This is going to create `deepspeech-tf` packages which should be just `deepspeech` (or PyPI will reject upload).

Reply:
@reuben yes, but I don't know how to fix that properly :/

Reply:
Pass build-flavor directly into the action and have a conditional there to set the correct project name?